backend/apis/parse-docket-pdf.api.js

```diff
@@ -34,9 +39,13 @@ app.post("/api/docket-pdfs", (req, res) => {
             if (processJson) {
               //comment this back in if you want to test flag json
               //res.send(generateFlagJson(parsePdfText(thePdf.text)));
-              res.send(parsePdfText(thePdf.text));
+              payload.push(parsePdfText(thePdf.text));
+              count--;
+              if (count === 0) res.send(payload);
```

Comment:
also, do we prefer using if statement here or something like this instead `count === 0 && res.send(payload)` instead?

Reply:
i think this is fine 👍